pages/multi/[address]/transaction/[transactionID].tsx

```diff
@@ -109,15 +114,18 @@ const transactionPage = ({
       setIsBroadcasting(true);
       setBroadcastError("");
 
+      if (!accountOnChain) throw new Error("Account on chain value missing.");
```

Comment:
why not use `assert` here?

Reply:
Good one. It does the same thing. First I wrote it manually, then I added `@cosmjs/utils`. Will update for consistency.